airbyte-analytics/src/main/java/io/airbyte/analytics/Deployment.java

```diff
@@ -26,16 +26,12 @@
 
 import com.google.common.base.Preconditions;
 import io.airbyte.config.Configs;
+import io.airbyte.config.Configs.DeploymentMode;
 import io.airbyte.config.Configs.WorkerEnvironment;
 import java.util.UUID;
 
 public class Deployment {
 
-  public enum DeploymentMode {
```

Comment:
moved this so that the configs module could access it. that is more in line with the project structure for other similar enums.